app/models/conference.rb

```diff
@@ -858,7 +858,7 @@ def cfp_set?
   # * +True+ -> If conference has a start and a end date.
   # * +False+ -> If conference has no start or end date.
   def registration_date_set?
-    !!registration_period && !!registration_period.start_date && !!registration_period.end_date
+    registration_period && registration_period.start_date && registration_period.end_date
```

Comment:
@Ana06 I backtracked `Conference#get_status calculates correct for new conference` test. It calls `get_status` message on conference object. Method `get_status` itself calls `registration_date_set?`. But I agree with 
>  The registration_period is an object
`.present?` would simply check weather it is `nil` or not. I have squased and rebased with 
` registration_period.present? && registration_period.start_date.present? && registration_period.end_date.present?`